synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -485,12 +486,68 @@ Option (ON by default) to make sure that if you ask for 50, you get 50% of the b
 	// Editing - Restrict Really-valued Handles to Top Right Quadrant
 	attach_label(pi.grid,_("Restrict really-valued handles to top right quadrant"), ++row);
 	pi.grid->attach(toggle_restrict_radius_ducks, 1, row, 1, 1);
-	toggle_restrict_radius_ducks.set_halign(Gtk::ALIGN_END);
+	toggle_restrict_radius_ducks.set_halign(Gtk::ALIGN_START);
 	toggle_restrict_radius_ducks.set_hexpand(false);
 	toggle_restrict_radius_ducks.set_tooltip_text("Restrict the position of the handle \
 (especially for radius) to be in the top right quadrant of the 2D space. Allow to set \
 the real value to any number and also easily reach the value of 0.0 just \
 dragging the handle to the left bottom part of your 2D space.");
+
+	attach_label_section(pi.grid, _("Edit in external"), ++row);
+
+	attach_label(pi.grid,_("Preferred image editor"), ++row);
+
+	//create a button that will open the filechooserdialog to select image editor
+	Gtk::Button *choose_button(manage(new class Gtk::Button(Gtk::StockID(_("Choose..")))));
+	choose_button->show();
+	choose_button->set_tooltip_text("Choose the preferred Image editor for Edit in external tool option");
+	
+	//create a function to launch the dialog
+	choose_button->signal_clicked().connect(sigc::mem_fun(*this,&Dialog_Setup::on_choose_editor_pressed));
+	pi.grid->attach(image_editor_path_entry, 1, row, 1, 1);
+	pi.grid->attach(*choose_button, 2,row,1,1);
+	image_editor_path_entry.set_hexpand(true);
+	image_editor_path_entry.set_text(App::image_editor_path);
+	
+}
+
+
+void
+Dialog_Setup::on_choose_editor_pressed()
+{
+	//set the image editor path = filepath from dialog
+	String filepath=image_editor_path_entry.get_text();
+	if(select_path_dialog("Select Editor", filepath ))
+		image_editor_path_entry.set_text(filepath);
+		App::image_editor_path = filepath;
+}
+
+bool 
+Dialog_Setup::select_path_dialog(const std::string &title, std::string &filepath)
+{
+	Gtk::FileChooserDialog *dialog = new Gtk::FileChooserDialog(*App::main_window,title, Gtk::FILE_CHOOSER_ACTION_OPEN);
+	dialog->set_transient_for(*App::main_window);
+	#ifdef WIN32
+	dialog->set_current_folder("C:\Program Files");
```

Comment:
Oh, sorry, it looks like GitHub incorrectly displayed my previous comment (for some reason it has a markdown-defined specific with displaying "\\" symbols) ^_^"

You need to use "\\\\" instead of "\\" here ^__^ - 
i.e. "C:\\\\Program Files"